libs/providers/vmware.py

```
@@ -1837,16 +1857,81 @@ def get_vm_or_template_networks(
+                        f"Found NIC: {device.deviceInfo.label if device.deviceInfo else 'unknown'}, backing type: {type(backing).__name__}"
+                    )
+
+                    if hasattr(backing, "network") and backing.network:
+                        network_names.add(backing.network.name)
+                        LOGGER.info(f"  -> Network: {backing.network.name}")
+                    elif hasattr(backing, "deviceName"):
+                        # Standard port group
+                        network_names.add(backing.deviceName)
+                        LOGGER.info(f"  -> Device name: {backing.deviceName}")
+                    else:
+                        LOGGER.warning(f"  -> No network name found in backing info")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)